docs/src/rules/no-param-reassign.md

```diff
@@ -6,13 +6,13 @@ further_reading:
 ---
 
 
-Assignment to variables declared as function parameters can be misleading and lead to confusing behavior, as modifying function parameters will also mutate the `arguments` object when not in strict mode (see [When Not To Use It](#when-not-to-use-it) below). Often, assignment to function parameters is unintended and indicative of a mistake or programmer error.
+Assignment to variables declared as `function` parameters can be misleading and lead to confusing behavior, as modifying `function` parameters will also mutate the `arguments` object when not in `strict` mode (see [When Not To Use It](#when-not-to-use-it) below). Often, assignment to `function` parameters is unintended and indicative of a mistake or programmer error.
```

Comment:
I think "function" shouldn't be in backticks in this context (and the rest of the document) as it doesn't refer to the `function` keyword but any kind of function.

Reply:
Sure, I'll fix it right away.

Then the backticks of `function` in the `description` should be removed too?

![image](https://github.com/user-attachments/assets/4f574fb9-1cdf-4b85-a32f-b1c7810d18de)

Reply:
> Then the backticks of `function` in the `description` section should be removed too?

I think yes.